apps/lab/src/app/page.tsx

```diff
@@ -1,13 +1,17 @@
 import { ViewWrapper } from './components';
+import { CardCustom } from '@1xdev/pablo'
 
 import styles from './home.module.css';
 
-export default function Page() {
+export default function Page({ children }: {children: React.ReactNode }) {
```

Comment:
I would suggest you to take the type to a specific named type:

```
type SomeType { 
 someField: type;
}
```

this is more readable.